tests/integration/test_spans.py

```diff
@@ -2497,109 +2228,26 @@ def summarize_outcomes(outcomes):
     if sample_rate == 1.0:
         spans = spans_consumer.get_spans(timeout=10, n=6)
         assert len(spans) == 6
-        outcomes = outcomes_consumer.get_outcomes(timeout=10, n=6)
-        assert summarize_outcomes(outcomes) == {(16, 0): 6}  # SpanIndexed, Accepted
+        outcomes = outcomes_consumer.get_outcomes(timeout=10, n=7)
+        assert summarize_outcomes(outcomes) == {
+            (16, 0): 6,  # SpanIndexed, Accepted
+            (15, 1): 2,  # Metric, Filtered
```

Comment:
@Dav1dde Can't quite say if this is right.

We likely started to emit metrics for usage and count_per_root all the time, which might itself be a bug, and they get filtered? What's the correct behavior here?

Reply:
Yeah then we should fix that, either not emit the metrics when they shouldn't be emitted or if they should be emitted make sure they aren't dropped/filtered.

I don't know if we need the span based metrics in circumstances where there is no span extraction, will need to find that out, but I don't think so 🤔

Reply:
I don't really get this failure. We get `disabled-namespace` outcomes for project 43 but I enable span ingestion for that project, no outcome gets emitted.

I'm not sure I understand what the failure is here, can you take a look please?

Reply:
That makes sense, because the Spans metrics namespace is gated behind the span ingestion feature flag [here](https://github.com/getsentry/relay/blob/fa3ab6d0467401fa99c03d5148820f4832651795/relay-server/src/services/processor/metrics.rs#L59-L68).

Reply:
Since project 43 is the root and has dynamic sampling removing things, the `organizations:standalone-span-ingestion` should be enabled on both projects as they are part of the same organization. Then the test is well set up and passes (as it should).